sonar-plugin-api/src/main/java/org/sonar/api/PropertyField.java

```diff
@@ -55,7 +55,10 @@
   /**
    * Indicative size of the field value in characters. This size is not validated, it is merely used by the GUI
    * to size the different input fields of a property set.
+   *
+   * @deprecated since 6.1, as it was only used for UI.
    */
+  @Deprecated
   int indicativeSize() default 20;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)